pkg/artifact/artifacts.go

```diff
@@ -37,8 +37,8 @@ func (q *Client) List(jobName string, buildID int) (artifacts []Artifact, err er
 }
 
 // GetArtifact download artifact using stream
-func (q *Client) GetArtifact(projectName, pipelineName string, buildID int, filename string) (io.ReadCloser, error) {
-	artifactURL := fmt.Sprintf("/job/%s/job/%s/%d/artifact/%s", projectName, pipelineName, buildID, filename)
+func (q *Client) GetArtifact(projectName, pipelineName string, isMultiBranch bool, branchName string, buildID int, filename string) (io.ReadCloser, error) {
```

Comment:
Consider the compitale, I prefer to provide a new function with the new parameters. For instance:

`func (q *Client) GetArtifactFromBranchPipeline(projectName, pipelineName string, isMultiBranch bool, branchName string, buildID int, filename string) (io.ReadCloser, error) {`

then, `GetArtifact` could call `GetArtifactFromBranchPipeline`.

The other developers might occur the compile errors if we just add some new parameters.

Reply:
good idea

Reply:
@LinuxSuRen done